org-formation/710-tgw/_tasks.yaml

```diff
@@ -111,6 +111,9 @@ TransitGatewayRoutes:
       # org-sagebase-scipoolprod
       internalpoolvpc:
         CIDR: "10.41.0.0/16"
+      # org-sagebase-synapsedw
+      vpc:
```

Comment:
the TgwSpokes needs to match in VPN config file as well, could you please update here as well?  https://github.com/Sage-Bionetworks-IT/organizations-infra/blob/master/org-formation/720-client-vpn/_tasks.yaml#L141